src/projectM-pulseaudio/qprojectM-pulseaudio.cpp

```diff
@@ -74,7 +74,7 @@ std::string read_config();
 #include <string.h>
 #include <errno.h>
 #include <sys/stat.h>
-#include <src/libprojectM/TestRunner.hpp>
+#include <TestRunner.hpp>
```

Comment:
Having test code directly inside a class isn't really clean IMO, but having the include dir fixed to this is at least a good step in the right direction :)